src/Loop.php

```diff
@@ -223,6 +223,15 @@ public static function supports($feature) {
     }
 
     /**
+     * Get the underlying loop handle.
+     *
+     * @return null|object|resource The loop handle the event loop operates on. Null if there is none.
+     */
+    public static function getLoopHandle() {
```

Comment:
*shrugs* call me lazy but I personally prefer to follow PSR-1 & -2. Despite their flaws (spaces instead of tabs, funky braces, and what not I don't agree with) it is the standard we currently have. Honestly I just follow PSR-1 & -2 blindly in my own projects because it is easy and I don't have to think about it. If we want to use our own preference I don't have strong opinions about it as long as we're consistent.